<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>팀 짜기</string>
     </attribute>
     <widget class="QPushButton" name="insert_soldier_btn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>520</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="text">
       <string>용병 투입</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchUserEdit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>9</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QListWidget" name="worker_list_widget">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>191</width>
        <height>461</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="main_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>롤 인력사무소 참가 명단</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>571</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QPushButton" name="insert_worker_btn">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>50</y>
        <width>71</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="text">
       <string>투입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_btn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QPushButton" name="make_team_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>571</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>출발 뿌우우</string>
      </property>
     </widget>
     <widget class="QLabel" name="player_cnt_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>54</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_group_btn">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>360</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>그룹
투입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insert_division_btn">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>440</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
       </font>
      </property>
      <property name="text">
       <string>분할
투입</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>인 게임</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
